cmd/kata-pkgsync/config.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 SUSE LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+obsprojects:
+    home:katacontainers:releases:
+
+        releases:
+            - master
+            - stable-1.8
+            - stable-1.7
+
+        architectures:
+            - x86_64
+
+packagecloud:
+    repo: katacontainers/testing
+
+distro-mapping:
+    openSUSE_Leap_15.0: opensuse/15.0
+    openSUSE_Leap_42.3: opensuse/42.3
+    Fedora_29         : fedora/29
+    Fedora_28         : fedora/28
```

Comment:
Nit: odd numeric ordering (would be good to sort the entire list alphabetically actually ;)